weblab/templates/datasets/dataset_form.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
```

Comment:
Should this file be called `dataset_create.html`?

Reply:
copied from entity_form so I don't know :-)

Reply:
Indeed, not sure why it's called that there either :) Especially as I think there's a line in the view specifying the template name anyway!